tds-itemselection-impl/src/main/java/tds/itemselection/loader/ISDBLoader.java

```diff
@@ -389,7 +389,7 @@ private void loadSegmentBlueprint (SingleDataResultSet bpTbl, SingleDataResultSe
       Double slope = new Double(record.<Float> get ("slope"));
       Double intercept = new Double(record.<Float> get ("intercept"));
       String adaptiveVersion = record.<String> get ("adaptiveVersion");
-      Integer abilityWeight = record.<Integer> get ("abilityweight");
+      Integer abilityWeight = (int)(float) record.<Float> get ("abilityweight");
```

Comment:
Should this be doing any rounding?  For example, if I have 4.6 would 4 be the expected value?